Clean up the docs (#152)
patternfly/react-data-view: packages/module/patternfly-docs/content/extensions/data-view/examples/DataView/DataView.md

For the toolbar, you can make use of the predefined `DataViewToolbar` component, which extends the PatternFly [toolbar](/components/toolbar) with the most common use cases. For more details, please refer to the [Toolbar](/extensions/data-view/toolbar) section. In case it does not fit your needs, you can also use your custom toolbar component.

Data can be presented using the predefined `DataViewTable` component, which is an abstraction above the PatternFly [table](/components/table). For more details, please refer to the [Table](/extensions/data-view/table) docs section. In the near future, we are also planning to introduce a predefined Card view component. If you have more specific needs to display data, you can pass your custom implementation as a `DataView` child.

```js file="./PredefinedLayoutFullExample.tsx"

```

## Advanced concepts

This section contains advanced features related to the `DataView` wrapping component and information to better understand how the data view works under the hood. 

### Data view context

> The **data view internal context** provides shared state to all sub-components. It lives inside the `DataView` component to store callbacks for the data selection (`onSelect`, `isSelected`, `isSelectDisabled`), internally computed `isSelectable` flag based on selection callbacks passed, and `activeState` of the data view (loading, error, etc.). Its values are set up through props of the `DataView` component.

Reviewer: ```suggestion
The **data view internal context** provides shared state to all sub-components. It lives inside the `DataView` component to store callbacks for the data selection (`onSelect`, `isSelected`, `isSelectDisabled`), internally computed `isSelectable` flag based on selection callbacks passed, and `activeState` of the data view (loading, error, etc.). Its values are set up through props of the `DataView` component.
```

Maybe it's just my lack of familiarity with the extension, but I'm struggling to understand this section or the value that this info adds. Is this just explaining how the DataView props are created? Do any of the examples demonstrate this? For me, this is a confusing section, but I'm also not a dev, so I know I'm not the target audience 😄

### Events context

> The **data view events context** provides a way of listening to the data view events from the outside of the component through the `DataViewEventsContext`. 
> 
> In order to give your components an access to the shared context, wrap them and your data view with the `DataViewEventsProvider`.

Reviewer: ```suggestion
The `<DataViewEventsContext>` provides a method of listening to the data view events from the outside of the component. 

In order to give your other UI components access to the shared context, wrap them and your data view with the `<DataViewEventsProvider>`.
```
I'm assuming "your components" means other components outside of the data view, but let me know if I'm wrong
